editor/test/actions.js

```diff
@@ -1,70 +1,474 @@
+			const blockA = {
+				uid: 'blockA',
+			};
+			const blockB = {
+				uid: 'blockB',
+			};
+			expect( actions.mergeBlocks( blockA, blockB ) ).toEqual( {
+				type: 'MERGE_BLOCKS',
+				blocks: [ {
+					uid: 'blockA',
+				}, {
+					uid: 'blockB',
+				} ],
+			} );
+		} );
+	} );
+
+	describe( 'autosave', () => {
+		it( 'should return AUTOSAVE action', () => {
+			expect( actions.autosave() ).toEqual( {
+				type: 'AUTOSAVE',
+			} );
+		} );
+	} );
+	describe( 'redo', () => {
+		it( 'should return REDO action', () => {
+			expect( actions.redo() ).toEqual( {
+				type: 'REDO',
+			} );
+		} );
+	} );
+	describe( 'undo', () => {
+		it( 'should return UNDO action', () => {
+			expect( actions.undo() ).toEqual( {
+				type: 'UNDO',
+			} );
+		} );
+	} );
+
+	describe( 'removeBlocks', () => {
+		it( 'should return REMOVE_BLOCKS action', () => {
+			const uids = [ 'uid' ];
+			expect( actions.removeBlocks( uids ) ).toEqual( {
+				type: 'REMOVE_BLOCKS',
+				uids: [
+					'uid',
+				],
+			} );
+		} );
+	} );
+	describe( 'removeBlock', () => {
+		it( 'should return REMOVE_BLOCKS action', () => {
+			const uid = 'uid';
+			expect( actions.removeBlock( uid ) ).toEqual( {
+				type: 'REMOVE_BLOCKS',
+				uids: [
+					'uid',
+				],
+			} );
+		} );
+	} );
+
+	describe( 'toggleBlockMode', () => {
+		it( 'should return TOGGLE_BLOCK_MODE action', () => {
+			const uid = 'uid';
+			expect( actions.toggleBlockMode( uid ) ).toEqual( {
+				type: 'TOGGLE_BLOCK_MODE',
+				uid: 'uid',
+			} );
+		} );
+	} );
+
 	describe( 'startTyping', () => {
 		it( 'should return the START_TYPING action', () => {
-			expect( startTyping() ).toEqual( {
+			expect( actions.startTyping() ).toEqual( {
 				type: 'START_TYPING',
 			} );
 		} );
 	} );
 
 	describe( 'stopTyping', () => {
 		it( 'should return the STOP_TYPING action', () => {
-			expect( stopTyping() ).toEqual( {
+			expect( actions.stopTyping() ).toEqual( {
 				type: 'STOP_TYPING',
 			} );
 		} );
 	} );
 
+	describe( 'toggleSidebar', () => {
+		it( 'should return TOGGLE_SIDEBAR action', () => {
+			const isMobile = true;
+			expect( actions.toggleSidebar( isMobile ) ).toEqual( {
+				type: 'TOGGLE_SIDEBAR',
+				isMobile: true,
+			} );
+		} );
+	} );
+
+	describe( 'setActivePanel', () => {
+		const panel = 'panelName';
+		expect( actions.setActivePanel( panel ) ).toEqual( {
+			type: 'SET_ACTIVE_PANEL',
+			panel: 'panelName',
+		} );
+	} );
+	describe( 'toggleSidebarPanel', () => {
+		it( 'should return TOGGLE_SIDEBAR_PANEL action', () => {
+			const panel = 'panelName';
+			expect( actions.toggleSidebarPanel( panel ) ).toEqual( {
+				type: 'TOGGLE_SIDEBAR_PANEL',
+				panel: 'panelName',
+			} );
+		} );
+	} );
+
+	describe( 'createNotice', () => {
+		const status = 'status';
+		const content = <p>element</p>;
+		it( 'should return CREATE_NOTICE action when options is empty', () => {
+			const result = actions.createNotice( status, content );
+			expect( result.type ).toEqual( 'CREATE_NOTICE' );
```

Comment:
[toMatchObject](http://facebook.github.io/jest/docs/en/expect.html#tomatchobjectobject) would be a perfect fit here and for the similar tests.

```js
expect( result).toMatchObject( {
    type: 'CREATE_NOTICE',
    notice: {
        status: 'status',
        content: <p>element</p>,
        isDismissible: true,
        id: expect.any( Number ),
    },
} );
```